src/views/Documentation/PageMeta.jsx

```diff
@@ -112,26 +147,38 @@ export default class PageMeta extends Component {
               [classes.pageNavigationWithoutPrevious]: !hasPreviousPage,
             })}>
             {hasPreviousPage && (
-              <Button
+              <PageTransition
                 onClick={this.handlePreviousPage}
-                color="secondary"
                 variant="extended"
                 aria-label="Previous Page"
                 className={classes.navigationButton}>
-                <ChevronLeftIcon />
-                {pageInfo.prev.title}
-              </Button>
+                <ArrowLeftIcon color="#000" className={classes.leftIcon} />
+                <div className={classes.rowFlex}>
+                  <div className={classes.leftButtonText}>
+                    <span className={classes.smallText}>Prev</span>
+                    <span className={classes.largeText}>
+                      {pageInfo.prev.title}
+                    </span>
+                  </div>
+                </div>
+              </PageTransition>
             )}
             {hasNextPage && (
-              <Button
+              <PageTransition
                 onClick={this.handleNextPage}
-                color="secondary"
                 variant="extended"
                 aria-label="Next Page"
                 className={classes.navigationButton}>
-                {pageInfo.next.title}
-                <ChevronRightIcon />
-              </Button>
+                <div className={classes.rowFlex}>
+                  <div className={classes.rightButtonText}>
+                    <span className={classes.smallText}>Next</span>
+                    <span className={classes.largeText}>
+                      {pageInfo.next.title}
+                    </span>
+                  </div>
+                  <ArrowRightIcon color="#000" className={classes.rightIcon} />
```

Comment:
The arrow icons should also be handled in `PageTransition`. You can create a required prop called `variant` that takes 2 possible values `prev` or `next`. You would then be able to deduce which icon to show based on the variant. For reference, you can look at how our `Button` component is created: https://github.com/taskcluster/taskcluster-web/blob/6649f4b4cc06713d8d5c8ee0c52c7176d8fb439d/src/components/Button/index.jsx#L38